### DISCUSSION TITLE
[Controls] Keyboard Movement Isn't Instant

### DISCUSSION THREAD

**Discussion by JCollins2048** [ihhub/fheroes2#8274] [General]
* Split from #6310

### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

In the original _Heroes II_, players could use the keyboard or Number Pad to move around instantly with a single press of a button.  In _fheroes2_, pressing a direction will, instead, **place a movement marker** on the map one tile away without actually moving the hero.  Pressing the same movement key will then initiate movement.

### Describe the possible solution

This was probably changed on purpose, but it would be nice if there was an option, somewhere, to change keyboard movement back to "instant" instead of "staggered" or "confirm". (Not sure what word describes it best.)

### Related issues

* #6315

**Comment by ihhub** [comment]
Moving this to discussion section as we can't have an option for every single small difference between fheroes2 and the OG.

**Comment by mlazdans** [comment]
I totally agree - movement should be without placing marker first, as with original. Or maybe I just can't find how to disable it?

Many thanks for the game, it is very playable.
